pkg/util/kubernetes/apiserver/apiserver.go

```diff
@@ -109,6 +119,12 @@ type APIClient struct {
 	timeoutSeconds int64
 }
 
+// chartUserValues is defined to unmarshall JSON data decoded from a Helm cart release into accessible fields
+type chartUserValues struct {
+	// user-defined values overriding the chart defaults
+	Config map[string]interface{} `json:"config,omitempty"`
+}
+
```

Comment:
I would suggest to move this logic inside the `flare` package. maybe create a sub package for kubernetes specific logic.
But the `chartUserValues` struct or `GetDs` `GetDeployement` is very specific to the flare use case. So can be better to not add it to the `apiserver` package

Reply:
👍 moved to the `flare` package in sub-package `manifests`